Libraries/LibWeb/EncryptedMediaExtensions/MediaKeySystemAccess.idl

```diff
@@ -15,20 +15,20 @@ enum MediaKeysRequirement {
 
 // https://w3c.github.io/encrypted-media/#dom-mediakeysystemconfiguration
 dictionary MediaKeySystemConfiguration {
-    DOMString                               label = "";
-    sequence<DOMString>                     initDataTypes = [];
+    Utf16DOMString                          label = "";
```

Comment:
The changes to port to UTF-16 should be squashed into the appropriate commits earlier in the PR.